backend/apps/events/models.py

```
@@ -18,6 +42,22 @@ class Event(models.Model):
+                    models.Q(event_type="internal", space__isnull=False, reservation__isnull=False)
+                    | models.Q(event_type="external")
+                ),
+            ),
+            models.CheckConstraint(
+                name="event_external_forbids_space_and_reservation",
+                condition=(
+                    models.Q(event_type="external", space__isnull=True, reservation__isnull=True)
+                    | models.Q(event_type="internal")
+                ),
+            ),
+        ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)